resteasy-core/src/main/java/org/jboss/resteasy/core/ResteasyProviderFactoryImpl.java

```diff
@@ -2361,6 +2367,7 @@ public void injectProperties(Object obj, HttpRequest request, HttpResponse respo
    }
 
    // Configurable
+   //TODO!! problem with listeners...

```

Comment:
Which problem?

Reply:
uh, stale comment. Basically, as the method returns a live collection, invocations to such method are to be treated the same as the methods that potentially change the ResteasyProviderFactory. That's already the case, though, in DelegateLazyResteasyProviderFactory (I didn't have that class yet when I wrote the comment)